frontend/vue.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    devServer:{
+        port : 8080,
+        proxy : {
+            '^/api': {
+                target: 'http://backend:3000',
```

Comment:
what led you to use a proxy to make the api calls?

Reply:
I admit, I was blindly following a tutorial of  a guy that had a similar setup. He sold me the idea saying later it would help manage between development and production environments. I guess it would make sense if I wanted to host the app somewhere and have a dynamic way of changing the base URL of API calls between developing locally and deploying to host, right? I suppose it doesn't make sense in this case. Removed it. Thanks.

Reply:
it would probably spare you some work configuring the CORS headers, but other than that would just be creating an additional layer that could be avoided